curriculum/challenges/chinese/05-back-end-development-and-apis/back-end-development-and-apis-projects/exercise-tracker.md

```diff
@@ -541,7 +541,7 @@ async(getUserInput) => {
 };
 ```
 
-你可以将 `from`、`to` 和 `limit` 参数添加到 `GET /api/users/:_id/logs` 请求检索任何用户的部分日志。 `from` 和 `to` 是 `yyyy-mm-dd` 形式的日期， `limit` 是希望返回的 log 数量。
+你可以将 `from`、`to` 和 `limit` 参数添加到 `GET /api/users/:id/logs` 请求检索任何用户的部分日志。 `from` 和 `to` 是 `yyyy-mm-dd` 形式的日期， `limit` 是希望返回的 log 数量。
```

Comment:
Changes to different languages happen in crowdin, you should change the English file only